docs/glossary.md

```diff
@@ -42,47 +42,43 @@ Extrinsic properties can affect the behavior of a function or method, but they a
+The property of a distributed system is that it will eventually come to some sort of consensus. A system stuck in an infinite loop would not be considered live, even if computations are taking place; a system that eventually provides a result, even if incorrect or it takes a long time, is considered to have liveness.
 
 ## Node explorer
 
-A tool which gives you information about a node, such as the latest blocks sealed, finalized, and the current chain state as known by that node.
+A tool that gives you information about a node, such as the latest blocks sealed, finalized, and the current chain state as known by that node.
 
 ## Nominated Proof of Stake (NPoS)
 
-A proof of stake system whereby nominators "lend" their stake to validators, as a show of faith in the good behavior of the validator.  Nominated proof-of-stake differs from delegated proof-of-stake in that nominators are subject to loss of stake if they nominate a bad validator; delegates are not subject to loss of stake based on the behavior of the validator.
+A proof of stake system whereby nominators "lend" their stake to validators as a show of faith in the good behavior of the validator. Nominated proof-of-stake differs from delegated proof-of-stake in that nominators are subject to loss of stake if they nominate a bad validator; delegates are not subject to loss of stake based on the behavior of the validator.
 
 ## Nominator
 
-Nodes which select a set of validators.  A certain amount of DOTs must be staked in order to do so, which may be lost if the validator behaves badly.  This forces nominators to carefully select validators.
+Nodes that select a set of validators. A certain amount of DOTs must be staked in order to do so, which may be lost if the validator behaves badly. This forces nominators to carefully select validators.
 
 ## NTP
 
-Networking protocol designed to synchronize the clocks of computers over a network. NTP allows you to synchronize the clocks of all the systems within the network. Currently it is required that validators' local clocks stay reasonably in sync, so you should be running NTP or a similar service
-
-## On-chain governance
-
-Governance of a blockchain which is controlled by mechanisms controlled by the blockchain.  On-chain governance allows for decisions can be made in a transparent manner.  Note that there are a variety of different algorithms for making these decisions, such as simple majority voting or identity-based quadratic voting.
+Networking protocol is designed to synchronize the clocks of computers over a network. NTP allows you to synchronize the clocks of all the systems within the network. Currently, it is required that validators' local clocks stay reasonably in sync, so you should be running NTP or a similar service.
```

Comment:
Mention that NTP stands for "Network Time Protocol"

Reply:
Fixed in branch fix-glossary-comments-issue6